renderer/components/settings-tab/language-switcher.tsx

```diff
@@ -4,6 +4,10 @@ import { useAtomValue, useSetAtom } from "jotai";
 const locales = {
   en: "English",
   ru: "Русский",
+  ja: "日本語",
+  zh: "漢語",
```

Comment:
```suggestion
  zh: "简体中文",
```
What's currently here is heavily associated of Traditional Chinese from Taiwan.